src/main/resources/lang/installer_es_es.properties

```diff
@@ -7,32 +7,38 @@ progress.done.server=Servidor instalado correctamente
-progress.generating.launch.jar.library=Generando JAR de lanzamiento del servidor: {0}
+progress.generating.launch.jar.library=Generando JAR de lanzamiento del servidor\: {0}
 progress.installing=Instalando Fabric Loader {0} en el cliente
 progress.installing.server=Instalando Fabric Loader {0} en el servidor
 prompt.exception=Error
 prompt.exception.caused.by=Causado por
-prompt.exception.occurrence=¡Ocurrió un error!
+prompt.exception.occurrence=¡Ocurrió un error\!
 prompt.exception.suppressed=Suprimido
 prompt.install=Instalar
-prompt.loader.version=Versión de Loader:
+prompt.loader.version=Versión de Loader\:
 prompt.loading.versions=Cargando versiones
 prompt.launcher.open.body=Por favor, cierra el lanzador de Minecraft antes de la instalación para asegurar un funcionamiento correcto.\n\n¿Quieres continuar igualmente?
 prompt.launcher.open.tile=El lanzador de Minecraft está abierto
-prompt.game.version=Versión de Minecraft:
+prompt.launcher.type.body=Fabric ha detectado dos instalaciones diferentes del Launcher de Minecraft. ¿En qué launcher deseas que Fabric se instale en?\n\n- Selecciona Tienda de Microsoft si estás jugando Minecraft desde la App de Xbox o la Tienda de Windows.\n- Selecciona Independiente si descargaste el Launcher de Minecraft directamente desde la página web de minecraft.net.\n\nSi no estás seguro, intenta priemro con la opción de Tienda de Microsoft. Siempre puedes correr el instalador de nuevo después.
+prompt.launcher.type.xbox=Tienda de Microsoft / Xbox
+prompt.launcher.type.win32=Independiente (Win32)
+prompt.game.version=Versión de Minecraft\:
 prompt.ready.install=Listo para instalar
-prompt.select.location=Elige carpeta de instalación:
+prompt.select.location=Elige carpeta de instalación\:
 prompt.server.info.jar=El JAR del servidor de Minecraft oficial es necesario para ejecutar Fabric
 prompt.server.info.command=Usa este comando para iniciar el servidor
 prompt.server.info.scipt=O genera scripts de lanzamiento
 prompt.server.jar=Descargar JAR del servidor
 prompt.server.jar.valid=JAR del servidor de la {0} válido encontrado
 prompt.server.jar.invalid=No se encontró un JAR del servidor de la {0} válido
+prompt.server.downloading=Descargando {0}/{1} MB
 prompt.server.generate=Generar
 prompt.server.overwrite=¿Estás seguro que quieres sobreescribir los scripts de lanzamiento existentes?
+prompt.server.launcher=Haz clic aquí para usar el lanzador del servidor independiente para una configuración más fácil
 prompt.install.successful.title=Instalado Correctamente
-prompt.install.successful=Fabric Loader {0} para {1} ha sido instalado correctamente.<br>Muchos mods también necesitan que descargues <a href="{2}">Fabric API</a> en tu carpeta de <a href="fabric://mods">mods</a>.
+prompt.install.successful=Fabric Loader {0} para {1} ha sido instalado correctamente.<br>Muchos mods también necesitan que descargues <a href\="{2}">Fabric API</a> en tu carpeta de <a href\="fabric\://mods">mods</a>.
```

Comment:
Still not quite right, not sure why after I disabled escaping 🤔

Reply:
Probably because `:` and `=` are key-value separators in the properties format, so it's being overly cautious.

Reply:
![Screenshot 2024-02-18 at 15 21 19](https://github.com/FabricMC/fabric-installer/assets/4324090/f288690c-9909-41c4-81f3-f8cf31656c0d)

Works as expected, so ill merge this 👍